packages/confirmation-modal/src/ConfirmationModal.spec.tsx

```diff
@@ -146,4 +146,16 @@ describe('packages/confirmation-modal', () => {
       expect(confirmationButton).toBeDisabled();
     });
   });
+
+  describe('submit is disabled when', () => {
+    test('"submitDisabled" prop is set', () => {
+      const { getByText } = renderModal({
+        open: true,
+        submitDisabled: true,
+      });
+
+      const confirmationButton = getByText('Confirm').closest('button');
+      expect(confirmationButton).toBeDisabled();
```

Comment:
can you also add that clicking the button doesn't cause the modal to be closed?

Reply:
👍